src/runAll.js

```diff
@@ -26,7 +26,7 @@ module.exports = function runAll(packageJson, config) {
   return new Promise((resolve, reject) => {
     sgf('ACM', (err, files) => {
       if (err) {
-        reject(err)
+        return reject(err)
```

Comment:
Just wondering, why is this needed?

Reply:
If we don't return on error, it executes the rest of the code. Which means it tries to do `files.map` even though it is `undefined`.

Reply:
Oh my bad. I assumed reject will implicitly return. We should add a test for that then.